src/components/lyne-cta-button/lyne-cta-button.tsx

```diff
@@ -24,6 +25,9 @@ export class LyneCtaButton {
   /** Label text to show on the button */
   @Prop() public label = 'Default button text';
 
+  /** Type of the button, like primary, secondary etc. */
+  @Prop() public type?: InterfaceButtonAttributes['type'] = 'primary';
```

Comment:
I'm wondering, whether we should better name that something like 'variant' since type is already used as an offical attribute of button (button, submit, reset).

Reply:
Good point! Completely ignored this one, will change it to `variant`